File: En C++ - Compilado/PalindromosReverse.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {
    string texto;
    int aux = 0, igual = 0;

    cout << "Ingrese la palabra a evaluar: ";
    getline(cin >> ws, texto);

    string reversa = texto;
    reverse(reversa.begin(), reversa.end());

    if(texto == reversa) {
        cout << "La palabra ingresada es palindromo!! :D" << endl;
    } else {
        cout << "La palabra ingresada no es palindromo!! :(" << endl;
    }

    return 0;
}

File: En Ruby - Interpretado/PalindromosReverse.rb
print "Ingrese la palabra que desea evaluar: "
texto = gets().chomp("\n")

textrevers = texto.reverse

if textrevers.downcase.eql?(texto.downcase)
  puts "La palabra ingresada si es palíndromo"
else
  puts "La palabra ingresada no es palíndromo"
end
